src/index.ts

```diff
@@ -1 +1,2 @@
 export { untilDestroyed } from './take-until-destroy';
+export { WithUntilDestroyed as AutoUnsubscribe } from './decorator';
```

Comment:
why as AutoUnsubscribe?

Reply:
My bad, it was a leftover after refactoring

Reply:
Fixed